components/database/couchdb-21/files/Makefile.openindiana

```diff
@@ -0,0 +1,21 @@
+   
+install:
+	if [ -d  $(DESTDIR)$(ETCDIR) ]; then    rm -rf $(DESTDIR)$(ETCDIR); fi
+	if [ -d  $(DESTDIR)$(MANDIR) ]; then    rm -rf $(DESTDIR)$(MANDIR); fi
+	if [ -d  $(DESTDIR)$(DOCDIR) ]; then    rm -rf $(DESTDIR)$(DOCDIR); fi
+	rm -rf   $(DESTDIR)$(RELDIR)/var	
+	mkdir -p $(DESTDIR)$(BINDIR)
+	mkdir -p $(DESTDIR)$(MANDIR)/man1/	
```

Comment:
Consider using macros from https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/make-rules/shared-macros.mk#L155, they might be helpful.